Sources/Plasma/NucleusLib/inc/plPipeline.h

```diff
@@ -357,6 +367,9 @@ class plPipeline : public plCreatable
     
     float fBackingScale = 1.0f;
     void SetBackingScale(float scale) { fBackingScale = scale; };
+    
+    std::shared_ptr<plDisplayHelper> fDisplayHelper;
```

Comment:
Probably want a better ownership model than shared pointer.

Reply:
Both the client and the pipeline need access to the display helper. I could roll this back so that the reference is owned by the client.